package.json

```diff
@@ -37,7 +37,7 @@
     "@angular/router": "10.0.8",
     "angular-cli-ghpages": "^0.6.2",
     "bootstrap": "4.3.1",
-    "core-js": "3.6.5",
+    "core-js": "2.6.5",
```

Comment:
Hey Gregg, what is the reason you downgraded to 2.6.5?

Reply:
That one was a mistake, I will fix it.

Reply:
Actually, I decided to keep that one the same as it was previously because the new version had errors when upgraded to it.